ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolSpec.java

```diff
@@ -72,8 +71,6 @@
 
   private final FeeMarket feeMarket;
```

Comment:
just a little bit confused with FeeMarket and BaseFeeMarket. there is the use of FeeMarket here but elsewhere it is used BaseFeeMarket instead. Why not always used the interface ?

Reply:
As part of the 1559 cleanup, `FeeMarket` is now a non-optional part of ProtocolSpec.  It includes TransactionPriceCalculator and minTransactionPriceInNextBlock, and is necessary for all of the forks.  There is a pre-1559 FeeMarket that is used for all ProtocolSpecs pre-London.  

`BaseFeeMarket` is a 1559-specific extension of FeeMarket and is used in all of the London-specific classes that need BaseFee constants and methods.